lib/cisco_node_utils/bgp_af.rb

```diff
@@ -226,6 +226,22 @@ def default_additional_paths_selection
       config_get_default('bgp_af', 'additional_paths_selection')
     end
 
+    # advertise_l2vpn_evpn
```

Comment:
in the cli we need to open those feature to set the property
feature vn-segment-vlan-based',
 'feature nv overlay',
 'nv overlay evpn